tests/conftest.py

```diff
@@ -291,6 +292,5 @@ def timezone(tzinfo):
 
 
 @pytest.fixture()
-def tmp_file(tmp_path):
-    with tmp_path / uuid4().hex as file:
-        yield file
+def tmp_file(tmp_path) -> Path:
+    return tmp_path / uuid4().hex
```

Comment:
just curious, what is this change about?

Reply:
Tests error out:

```python
    def tmp_file(tmp_path) -> Path:
>       with tmp_path / uuid4().hex as file:
E       TypeError: 'PosixPath' object does not support the context manager protocol
```